packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -12,15 +12,24 @@ import { __ } from '@wordpress/i18n';
 
 function KeyboardShortcuts() {
 	// Shortcuts Logic
-	const { clientIds, rootBlocksClientIds } = useSelect( ( select ) => {
-		const { getSelectedBlockClientIds, getBlockOrder } = select(
-			'core/block-editor'
-		);
-		return {
-			clientIds: getSelectedBlockClientIds(),
-			rootBlocksClientIds: getBlockOrder(),
-		};
-	}, [] );
+	const { clientIds, rootBlocksClientIds, rootClientId } = useSelect(
+		( select ) => {
+			const {
+				getSelectedBlockClientIds,
+				getBlockOrder,
+				getBlockRootClientId,
+			} = select( 'core/block-editor' );
+			const selectedClientIds = getSelectedBlockClientIds();
+			const normalizedClientIds = castArray( selectedClientIds );
```

Comment:
Looking at [`getSelectedBlockClientIds`](https://github.com/WordPress/gutenberg/blob/77171699b76f8ebc3b0d552f0d84fc2ec650a231/packages/block-editor/src/store/selectors.js#L635-L683), it always returns an array. Any reason to array-cast into a new variable?

Reply:
You're right! Thanks! I removed it.